src/core/strvector.c

```diff
@@ -333,6 +355,44 @@ igraph_error_t igraph_strvector_append(igraph_strvector_t *to,
     return IGRAPH_SUCCESS;
 }
 
+/**
+ * \ingroup strvector
+ * \function igraph_strvector_merge
+ * \brief Merges a string vector into another, clearing the first one.
```

Comment:
The word "first" is potentially confusing here because the order of the _arguments_ is "to" and then "from" so the vector being cleared is the _second_ argument, but the _first_ vector in the sentence. It would be good to rephrase this somehow to avoid confusion. Something like: merges a _source_ vector into a _destination_ vector and clears the _source_ afterwards.

Reply:
How about this? _"Merges a string vector into another without reallocating the contents"_  This makes it clear what the major difference is from `append`. Or "... without reallocating individual strings".

Or maybe: "Moves the contents of a string vector into another" or "... to the end of another." The word "move" implies that there is no copying.

Reply:
good! Feel free to merge if this is fixed.

Reply:
I mean, I vote for the "moves the contents" variant.